core/os/android/adb/commands.go

```diff
@@ -406,15 +406,16 @@ func (b *binding) QueryAnglePackageName(ctx context.Context) (string, error) {
 	}
 	// ANGLE supported, so check for installed ANGLE package
 	// Favor custom installed package first, followed by default system package
-	custom := "com.chromium.angle"
+	//  org.chromium.angle is first because it's the most recent & preferred name
+	custom := [2]string{"org.chromium.angle", "com.chromium.angle"}
 	system := "com.google.android.angle"
 	// Check installed packages for ANGLE package
 	packages, _ := b.InstalledPackages(ctx)
 	if pkg := packages.FindByName(custom); pkg != nil {
-		return custom, nil
+		return pkg.Name, nil
 	}
```

Comment:
You need a loop here now that `custom` is a slice, no?

Reply:
Yes, I misread FindByName thinking that it took a slice and processed those strings in a loop, but it only takes a string and processes the package names in a loop.
I think the easier and cleaner thing to do in the long-term is just switch to the org.chromium.angle name only going fwd. I'm pushing that now so the com.chromium.angle name should die quickly and better to not keep the incorrect name alive.